File: myExpenses/src/main/java/org/totschnig/myexpenses/retrofit/RoadmapService.java
package org.totschnig.myexpenses.retrofit;

import java.util.List;

import retrofit2.Call;
import retrofit2.http.Body;
import retrofit2.http.GET;
import retrofit2.http.POST;

public interface RoadmapService {
  @GET("issues")
  Call<List<Issue>> getIssues();
  @POST("votes")
  Call<Void> createVote(@Body Vote query);
}

File: myExpenses/src/main/java/org/totschnig/myexpenses/retrofit/RoadmapService.kt
package org.totschnig.myexpenses.retrofit

import retrofit2.Call
import retrofit2.Response
import retrofit2.http.Body
import retrofit2.http.GET
import retrofit2.http.POST
import retrofit2.http.Query

interface RoadmapService {
    @GET("issues")
    suspend fun issues(): Response<List<Issue>>

    @POST("votes")
    suspend fun createVote(@Query("sha") sha: String, @Body query: Vote): Response<Unit>
}